core/pom.xml

```diff
@@ -12,10 +12,29 @@
 
 	<properties>
 		<spring.version>4.3.25.RELEASE</spring.version>
-		<resteasy.version>3.0.12.Final</resteasy.version>
 		<j2v8.version>4.6.0</j2v8.version>
+		<jackson.version>1.1.1</jackson.version>
 	</properties>
 	<dependencies>
+
+	<dependency>
```

Comment:
Isn't this dependency indirectly included because of 'cxf-rt-rs-client'?

Reply:
this class doesn't exist in cxf-rt-rs-client': org.codehaus.jackson.jaxrs.JacksonJsonProvider